### PR TITLE
use planner-version instead of planner_version

### PR DESCRIPTION
**Pull request by systay**
Signed-off-by: Andres Taylor <andres@planetscale.com>

### CODE REVIEW

**Review by ajm188 — COMMENTED** [review]
i have bad news (😅 🙈): _all_ of these should be double-dash–prefixed (e.g. `-planner-version` => `--planner-version`, and every other flag in that table in vtgate.md)

**Comment by ajm188** [comment]
> i have bad news (😅 🙈): _all_ of these should be double-dash–prefixed (e.g. `-planner-version` => `--planner-version`, and every other flag in that table in vtgate.md)

something close to `sed 's/^| -/| --/` should do the trick (notably not idempotent, but you could enhance with negative lookahead/capture groups, etc)

**Review by ajm188 — APPROVED** [review]

File: content/en/docs/14.0/reference/compatibility/gen4-and-v3-compatibility.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ To ensure `Gen4` produces plans that, once executed, give us the same results as
 a small test tool that runs queries using both `Gen4` and `V3` planners and compare their results. If the results we got are different
 from the two planners, the query will fail and the difference will be printed in VTGate's logs as a warning.
 
-This tool is enabled by the use of a new planner: `Gen4CompareV3`, to use it, we must start VTGate with the `-planner_version` 
+This tool is enabled by the use of a new planner: `Gen4CompareV3`, to use it, we must start VTGate with the `-planner-version`
```
Comment (deepthi) [inline content/en/docs/14.0/reference/compatibility/gen4-and-v3-compatibility.md:12]: needs to be `--`

File: content/en/docs/14.0/reference/programs/vtexplain.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The following parameters apply to `mysqlctl`:
 | Name                                 | Type | Definition |
 |:-------------------------------------| :--------- | :---------------------------------------------------- |
 | -output-mode                         | string | Output in human-friendly text or json (default "text") |
-| -planner_version                     | string | Sets the query planner version to use when generating the explain output. Valid values are V3 and Gen4 (default "Gen4") |
+| -planner-version                     | string | Sets the query planner version to use when generating the explain output. Valid values are V3 and Gen4 (default "Gen4") |
 | -normalize                           |  | Whether to enable vtgate normalization (default false) |
 | -shards                              | int | Number of shards per keyspace (default 2) |
 | -replication-mode                    | string | The replication mode to simulate -- must be set to either ROW or STATEMENT (default "ROW") |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtexplain.md:35]: All of these too.
Btw, the changes to vtgate.md are duplicated in #1046. Two ways to resolve:
- only change the compatibility doc in this PR and let #1046 handle the rest. 
- also change vtexplain.md in this PR and let #1046 do a merge/rebase.

**Comment by mattlord** [comment]
@systay I'm sorry for the hassle 😢 Want me to try and incorporate this into #1046 as well?

https://github.com/vitessio/website/pull/1046/commits/923a6811ec6aa883ea15e21987669287e51f480f

**Comment by deepthi** [comment]
Included in #1046

**Pull request closed without merging**